tests/integration_tests/conftest.py

```
@@ -10,15 +10,16 @@
 
 import pytest
 
-# Add project path
-project_root = Path(__file__).parent.parent
-sys.path.insert(0, str(project_root))
+# Add project path (repo root + src)
+repo_root = Path(__file__).resolve().parents[2]
+src_root = repo_root / "src"
+sys.path.insert(0, str(src_root))
 
 import pyseekdb  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)